Enemy

jsuk10 opened the pull request:
make a EnemyMove.cs, Rabbit.prefab

apply Move.cs and I made sure it works

Move by referring to "wayPoint".
It moves through the current index, and if the current index is 0 or equals the length of the array, it moves by referencing in reverse order of progression.

적움직임 스크립트, 토끼 프리팹을 만들었음

적움직입 스크립트가 잘 작동하는것을 확인하였고

"wayPoint"를 참조하여 이동합니다.
현재 인덱스를 통해 이동하며 현재 인덱스가 0이거나 배열 길이와 같으면 진행의 역순으로 참조하여 이동합니다.

ManaBingsu commented:
Player.meta 의 guid를 player의 guid로 변경, 머지하였습니다.

jsuk10 committed:
chore : init & test

jsuk10 committed:
chore : init 2

using namespace enemy

jsuk10 committed:
feat : MobMove

make a function
LookTarget - 대상방향 바라보는 함수
PhysicalCalculation - 물리적 연산을 하는 함수
MoveToWayPoint - 웨이포인트로 이동하는 함수
ChangeTarget - 타겟을 바꿔주는 함수

jsuk10 committed:
feat : Rabbit Make Prefabs

make a Rabbit Prefabs and attach EnemyMove

jsuk10 committed:
chore : remove debug.log

ManaBingsu reviewed (changes requested):
Naming convention이 지켜지지 않았습니다.

jsuk10 committed:
style : Modified according to Naim convention

jsuk10 committed:
feat : Implemented to be able to calculate and view angles

"sixAxis Look" you can choose whether to see the angle or change only the x-axis.
"_isYAxisReverce" up and down reversal can also be controlled through -for Gumbang

"sixAxis Look"값을 이용해 앵글을 볼지x축만 바꿀지 선택 할 수 있음
"_isYAxisReverce"를 통해 상하 반전도 제어 가능 - 굼뱅이용

jsuk10 committed:
fix : outOfRange

Exception handling beyond index range
인덱스 범위 넘어가는거 예외처리 해줌

jsuk10 committed:
chore : Make a example Scenes
